### PR TITLE
Claims button opens the inventory tab on click

### PR DESCRIPTION
**Pull request by mavisakalyan**
Added a new state so whenever we need it we can specify a tab to open.

## Describe your changes

- Added another state called "openedTab" whenever specified will open the tab that is set there.
- Now whenever the claims button is pressed when the character menu opens the inventory tab will be shown.
- if the openedTab is not specified the default tab will be opened which is Account for now.

## What are the steps for a QA tester to test this pull request?

- Open character menu switch through the tabs of the equipment box.
- Then close and press inventory button to open the character menu with the inventory tab open.

## Issue ticket number and link

https://github.com/webaverse/app/issues/3387

## Screenshots and/or video

https://user-images.githubusercontent.com/55106546/183299215-33b512fb-8140-4d93-8dcb-e74c0a7c924e.mov

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I am not adding any irrelevant code or assets
- [x] I am only including the changes needed to implement the change
- [x] I have playtested and intentionally tried to find error cases but couldn't

### CODE REVIEW

File: src/components/general/equipment/Equipment.jsx [hunk 1]
```diff
@@ -406,6 +406,28 @@ export const Equipment = () => {
                     abortController.abort();
                 };
             }
+            
+            if(state.openedTab) {
+                let selectedTab;
+                switch (state.openedTab) {
+                case 'Inventory':
```
Comment (avaer) [inline src/components/general/equipment/Equipment.jsx:413]: Missing indent.

**Review by avaer — CHANGES_REQUESTED** [review]
This shouldn't be updating submodules under `packages/` and `package-lock.json`.

**Comment by alisaad673** [comment]
Needs conflict resolution

File: src/components/general/equipment/Equipment.jsx (hunk 1 above)
Comment (mavisakalyan) [reply src/components/general/equipment/Equipment.jsx:413]: Got it. Will keep an eye on that next time.

**Comment by avaer** [comment]
Not resolved and cannot be tested.

**Pull request closed without merging**